### ISSUE TITLE
Wiki request - rewrite and simplify explanation of Roman Numerals

### ISSUE THREAD

**Issue by bugron** [FreeCodeCamp/wiki#356]
Original issue: https://github.com/FreeCodeCamp/FreeCodeCamp/issues/3414

http://www.mathsisfun.com/roman-numerals.html

**Comment by thefreezer** [comment]
I believe that http://www.mathsisfun.com/roman-numerals.html already provides a well-written and clear explanation on Roman Numerals. It will be harder to simplify it further.

**Comment by raisedadead** [comment]
@thefreezer I agree. How ever it would be cool if we write something on the similar lines. Its easier for campers to find articles in the FCC wiki.

You may take the article as it is, **BUT, BUT, BUT ensure you give credits to the original author 100%** we do not want to plagiarize anything.
